### PR TITLE
Add Textile language

### PR DESCRIPTION
**Pull request by Golmote**
Preview of the examples:

prism-textile

### CODE REVIEW

File: components/prism-textile.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+	// Modifiers highlighting
+	Prism.languages.textile['phrase'].inside['block'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+	Prism.languages.textile['phrase'].inside['list'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+	Prism.languages.textile['phrase'].inside['table'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+	Prism.languages.textile['phrase'].inside['inline'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+	Prism.languages.textile['phrase'].inside['link'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+	Prism.languages.textile['phrase'].inside['image'].inside['modifier'].inside = Prism.util.clone(modifierTokens);
+
+	// Allow some nesting
+	Prism.languages.textile['phrase'].inside['inline'].inside['bold'].inside = {
```
Comment (LeaVerou) [inline components/prism-textile.js]: Wow, a lot of repetition here. Why not define one object and clone it?

**Comment by Golmote** [comment]
@LeaVerou It should be more DRY now. Does it look good to you?

**Comment by LeaVerou** [comment]
Is cloning really needed? So what if they refer to the same object? It’s not like it’s mutated at any point.

**Comment by Golmote** [comment]
Well, I do not mutate it in any way but a plugin could. The last time I forgot to clone duplicated parts of the grammar, bad things happened (remember #469), so I thought prevention was better than cure.

**Pull request merged**